x/ethbridge/client/cli/tx.go

```diff
@@ -259,3 +259,35 @@ func GetCmdUpdateWhiteListValidator(cdc *codec.Codec) *cobra.Command {
 		},
 	}
 }
+
+// GetCmdUpdateCethReceiverAccount is the CLI command for update the validator whitelist
```

Comment:
It seems like this comment should say:

`// GetCmdUpdateCethReceiverAccount is the CLI command to update the sifchain account that receives the ceth proceeds`